### PR TITLE
Cleanup network section

### PR DESCRIPTION
**Pull request by pchaussalet**
Fix  network-interface ip address management.
Move network section to the new UI architecture:
- Only one cascading list for the whole application
- Same structure for all sections (overview, entries, settings)
- Definition of sections using a mjson file instead of hardcoded in different places
- Automatic loading of first level elements using SectionService

### CODE REVIEW

File: ui/sign-in.reel/sign-in.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ var SignIn = exports.SignIn = AuthorizationPanel.specialize({
 
                     //FIXME: kind of hacky
                     self.application.dispatchEventNamed("userLogged");
-                    self.application.section = self._getSection();
+//                    self.application.section = self._getSection();
```
Comment (thibaultzanini) [inline ui/sign-in.reel/sign-in.js]: why this line is just commented?

File: core/dao/abstract-dao.js [hunk 2]
```diff
@@ -19,6 +19,14 @@ exports.AbstractDao = Montage.specialize({
         }
     },
 
+    get: {
+        value: function() {
+            return this.list().then(function(entries) {
+                return entries[0];
```
Comment (thibaultzanini) [inline core/dao/abstract-dao.js:25]: always an empty array right? (from montage data I believe)

File: core/dao/section-settings-dao.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/**
+ * @module core/dao/section-settings-dao
+ */
+var AbstractDao = require("core/dao/abstract-dao").AbstractDao;
+/**
+ * @class SectionSettingsDao
+ * @extends AbstractDao
+ */
+exports.SectionSettingsDao = AbstractDao.specialize(/** @lends SectionSettingsDao# */ {
+    constructor: {
```
Comment (thibaultzanini) [inline core/dao/section-settings-dao.js]: could be removed

**Review by thibaultzanini — COMMENTED** [review]
LGTM just a few questions.

**Comment by thibaultzanini** [comment]
@pchaussalet Branch as conflicts

File: core/dao/abstract-dao.js (hunk 2 above)
Comment (pchaussalet) [reply core/dao/abstract-dao.js:25]: AFAIK, it is (we'd have issues if it weren't).

File: core/dao/section-settings-dao.js (hunk 3 above)
Comment (pchaussalet) [reply core/dao/section-settings-dao.js]: Done

**Comment by thibaultzanini** [comment]
LGTM

**Pull request merged**